README.md

```diff
@@ -169,14 +172,142 @@ The command above will create three pods associated to the existing `imagestream
+Writing manifest to image destination
+Storing signatures
+d184fda91f0aa5c6deed433f984bd393754f707a285445a83637aaa13b8b7e86
+Spawning a container 'toolbox-root' with image 'registry.redhat.io/rhel9/support-tools'
+Detected RUN label in the container image. Using that as the default...
+57305547f44861347e453b535fd59a56b4d9c0a3f472b7f1d86c2f246c94a5ea
+toolbox-root
+Container started successfully. To exit, type 'exit'.
+[root@master-0 /]#
+```
+
+Finally we can collect all required config/logs
+
+```bash
+[root@master-0 /]# sos report -k crio.all=on -k crio.logs=on
+<snip output>
+  Finished running plugins
+Creating compressed archive...
+
+Your sosreport has been generated and saved in:
+  /host/var/tmp/sosreport-master-0-2023-11-15-mvgdmxo.tar.xz
+
+ Size 82.68MiB
+ Owner  root
+ sha256 81b356f0069b4bc35dc4ae016e2c25369c9cefa214c75973cde8b8470ffa4516
+
+Please send this file to your support representative.
+
+[root@master-0 /]#
+```
+
+You can also add the flag `--all-logs` to the `sos` command to retrieve further configuration file and logs
+
+For more information see the official OCP documentation [here](https://docs.openshift.com/container-platform/4.13/support/gathering-cluster-data.html)
+
+#### EDPM compute nodes
+
+First we need to gather compute node info from `OpenstackDataPlaneNodeSet` resources.
+
+```bash
+$ oc get openstackdataplanenodesets -o name
+openstackdataplanenodeset.dataplane.openstack.org/openstack-edpm-ipam
+$ oc get openstackdataplanenodeset.dataplane.openstack.org/openstack-edpm-ipam -o json | jq -r '.spec.nodes[] | [.hostName, .ansible.ansibleHost] | @csv'
+"edpm-compute-0","192.168.122.100"
+"edpm-compute-1","192.168.122.101"
+"edpm-compute-2","192.168.122.102"
+```
+
+Then you have to login into nodes via ssh. In the `OpenstackDataplaneNodeSet` resource you can find the user used by ansible and the private ssh, that we can extract from the secret.
+
+```bash
+# user
+$ oc get openstackdataplanenodeset.dataplane.openstack.org/openstack-edpm-ipam -o json | jq .spec.nodeTemplate.ansible.ansibleUser
+"zuul"
+# private ssh key
+$ oc get openstackdataplanenodeset.dataplane.openstack.org/openstack-edpm-ipam -o json |jq .spec.nodeTemplate.ansibleSSHPrivateKeySecret
+"dataplane-ansible-ssh-private-key-secret"
+# save private ssh key to a file and fix its permissions
+$ oc get secret/dataplane-ansible-ssh-private-key-secret -o go-template='{{ index .data "ssh-privatekey" | base64decode }}'  > ~/.ssh/compute.key
+$ chmod 0600 ~/.ssh/compute.key
+```
+
+Finally login into compute node
+
+```bash
+$ ssh -i ~/.ssh/compute.key zuul@192.168.122.100
+Register this system with Red Hat Insights: insights-client --register
+Create an account or view all your systems at https://red.ht/insights-dashboard
+Last login: Wed Nov 15 10:02:11 2023 from 192.168.111.1
+[zuul@compute-0 ~]$
+```
+
+And launch the `sos` command just like we did in the other example, but with different flags.
+
+```bash
+sudo sos report -p system,virt,openstack_edpm
```

Comment:
-1: This is insufficient, we also need storage related plugins such as `block,iscsi,scsi,multipath,devicemapper,devices,lvm2,nfs,cifs,nvme,udev`

Reply:
ack!